setup.pl

```diff
@@ -560,7 +562,14 @@
 		print START "exec '$wadir/miniserv.pl' $config_directory/miniserv.conf\n";
 		}
 	close(START);
-	$start_cmd = "$config_directory/start";
+
+	# Define final start command
+	if ($upgrading) {
```

Comment:
N: Start script must be conditional. In case of install - it must always call `start` command and in case of upgrade it must always call conditional `.post-install`.